apps/portal/controllers/error-pages/error401.html

```diff
@@ -17,7 +17,7 @@
 <i class="icon-unlink error-icon"></i>
 <h1>Error 401</h1>
 <h3>You do not have permission to access this page.Please contact your administrator and request permission.</h3>
-<h4><a href="/portal/dashboards">Go to Portal</a></h4>
+<h4><a href="/portal">Go to Portal</a></h4>
```

Comment:
Localization cannot be done with the current implementation since the page is HTML.